src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/utils/validation.ts

```diff
@@ -42,19 +44,28 @@ export function createValidator(schema: any): ISchemaValidator {
     strictTuples: false,
     unicodeRegExp: false,
     code: { es5: true },
-  });
+  };
+  let ajv: Ajv | Ajv2020;
+  let rootElementPath;
+  if (schema.$schema?.includes('2020-12')) {
```

Comment:
Should any future draft be treated as the older version?

Reply:
If `ajv `adds support for future drafts we would have to add logic for handling this.